src/mpid/ch4/include/mpidpre.h

```diff
@@ -209,6 +210,7 @@ typedef struct MPIDIG_req_t {
     void *buffer;
     MPI_Aint count;
     MPI_Datatype datatype;
+    MPL_pointer_attr_t buf_attr;
```

Comment:
This causes the compile-time assertion about the size of the extended request object to fail in Jenkins tests. Do we need the whole attribute struct or can we get by with just the type?